platform/windows/windows.cmake

```diff
@@ -0,0 +1,3 @@
+if(MSVC)
+	include(${CMAKE_CURRENT_LIST_DIR}/windows.msvc.cmake)
```

Comment:
I would not split the file per compiler.

Reply:
I was hoping to have MingGW support in the future. If so, it would make sense? Or it's better even with MinGW support to have just one file?

Reply:
For Qt the changes between MSVC and MinGW are quite small. Also it will probably cause less duplication of code.

Reply:
Great. So I will migrate windows.msvc.cmake to windows.cmake, and when time comes, make the required changes to have it compatible with MinGW.